config/crd/bases/scheduling.sigs.k8s.io_elasticquotas.yaml

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
The new 3 yamls actually duplicate the crd files that are placed in the plugin folder (named crd.yaml). We want to avoid this kind of duplication. Can you create an issue to track a follow-up item: du-duplicate the CRD yaml definitions.

Reply:
This is not a duplication. This folder `config/crd/bases` is the default place for any CRD generated via `kubebuilder`. We can add `symlink `to any other folder you may see, i.e. `manifest`.

Reply:
I understand this is the default place controller-gen generates the CRD manifests. My point is basically as what you said, to have one single place as the source of truth of these CRD yamls.

Reply:
I did that by changing the `./verify-crdgen.sh` to check on `config/crd/bases` instead of `manifest` folder.

Do you want me to add a`symlink`?

Reply:
https://github.com/kubernetes-sigs/scheduler-plugins/pull/340/files#diff-4e3ddc507e7011467ddc0a1168a0eb8355821fe18f934b3a9042fb6ceef37cacR35

Reply:
it should be good. And yes, add a symbolic link from manifests/*/crd.yaml to config/crd/bases, so we don't have two copies.

Reply:
I'll have to put the annotation for the `KEP` link manually as nothing in `kubebuilder` can help to add it programmatically.